docs/src/user-guide/formatters/index.md

```diff
@@ -83,28 +97,48 @@ function addOne(i) {
 ```
 
 ### junit
 
+TODO: quick overview
```

Comment:
Another one that you use when you want to pass the data to a system that expects a specific format.

Context: https://github.com/eslint/eslint/issues/16

Reply:
Meant for use with the [JUnit Jenkins plugin](https://plugins.jenkins.io/junit/).